Dockerfile

```diff
@@ -21,10 +21,13 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 
 FROM alpine:3.15
```

Comment:
Don't we need to delete the alpine line here?

Reply:
Addressed in f906edd.